docs/pages/getting-started.mdx

```diff
@@ -1,3 +1,11 @@
-# Get started
+# Getting Started
 
-Hello world!
\ No newline at end of file
+Welcome to InProcess! This guide will help you get started with our protocol.
+
+## Quick Start
+
+1. Visit our website at [https://www.inprocess.fun/](https://www.inprocess.fun/)
+2. Connect your wallet
```

Comment:
We want to avoid technical / onchain terms to be more accessible to mainstream creators.
```suggestion
2. Sign In
```